src/components/MediaList/MediaList.tsx

```diff
@@ -223,8 +244,7 @@ export const MediaListItemMeta = ({ items = [] }: MediaListItemMetaProps) => {
   const attributionMeta = attributionItems.map((item) => `${item.label}: ${item.description}`).join(", ");
 
   return (
-    // eslint-disable-next-line react/no-unknown-property
-    <ul property="cc:attributionName" content={attributionMeta}>
+    <ul content={attributionMeta}>
```

Comment:
Ser at content her er heile innholdet i lista. Er det for skjermlesere?

Reply:
det er bare en content category, noe som er igjen fra når komponenten ble flyttet. tror ikke vi egentlig bruker dette så mye andre steder så kan sikkert fjernes om det er ønskelig

Reply:
Eg aner ikkje om det er ønskelig. Veit nokon andre om content leses opp av skjermlesere? @rauboti eller @Jonas-C kanskje?

Reply:
Tviler sterkt. Det er egentlig en attributt som kun eksisterer på `meta`, såvidt jeg vet. Vil tro at dette er CC som har bestemt seg for å ta den i bruk på en spesiell måte, om det har noe betydning i det hele tatt.

Reply:
Tror ikke jeg så dette i noen av eksemplene fra CC men det var ikke mange eksempler med `ul`. `Property` som var der før var feil plassert, så jeg tenker kanskje like greit å bare fjerne `content ` attributt også 🤷‍♂️

Reply:
Som sagt så står ikke det veldig mye konkret informasjon om det kreves, men ser at `content` attributten oftest er brukt i sammenheng der du forenkler innhold for maskiner. Måten den er brukt her på gir ikke så mye mening så derfor har jeg bestemt at vi fjerner det